### PR TITLE
Add 2 tier reviews for x% of PRs. Assigns to owner

### PR DESCRIPTION
**Pull request by eleddy**
This will assign a PR approved for merge to a component owner X% of the time, where x < 50%. A little hacky but should get the job done. We could make a separate label for further review if this is confusing to people.

### CODE REVIEW

**Comment by oahayder** [comment]
Where are the tests?!?! Lol jk. Ship it!

File: bin/issue_poller.py [hunk 1]
```diff
@@ -3,10 +3,12 @@
 from src.data_access.repository_access import GitHubController
 from time import sleep
 from conf.config import LABEL_TEAM_MAP, TARGET_REPOS
+from conf import config
 
-class IssuePoller():
+
+class IssuePoller(object):
```
Comment (oahayder) [inline bin/issue_poller.py]: Why did you need to add object here?

Comment (eleddy) [inline bin/issue_poller.py]: Python 2 says we should be explicit about new style classes. Its a good habit, and your editor should yell if you don't. See 2nd answer at http://stackoverflow.com/questions/4015417/python-class-inherits-object

**Comment by eleddy** [comment]
haha test free is the way to be! Do you think this will be an ok method? Goes for second review after Merge? Also, how do I get this puppy redeployed after?
